avo-hq/avo
# feature: automate tailwind installation to Avo

# Description


Fixes https://github.com/avo-hq/avo/issues/1188

# Checklist:




## Manual review steps


1. Create a new rails app that uses sprockets
1. Run `bin/rails generate avo:tailwindcss:install`
2. Observe that it ejected the `_head` template, created the `avo.tailwind.css` file, added the file to the `_head` template, and updated the `Procfile.dev` with the `avo_css: bin/rails tailwindcss:watch` command.

Manual reviewer: please leave a comment with output from the test if that's the case.
